### PR TITLE
Annotations - DO NOT MERGE

### PR DESCRIPTION
**Pull request by benjamine**
testing using a PR to do inline comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1 @@
-Shopbeam Overview
------
-
-Shopbeam provides tools for advertisers and publishers to create and serve display advertisements which load product detail windows on-site. These on-publisher site product detail modal windows can be used to add products to a cross site shopping cart and purchase these products without clicking away from host/publisher website.
```
Comment (benjamine) [inline README.md:4]: "Shopbeam tools add e-commerce capabilities to sites across the web (eg. fashion blogs, magazines, brand sites) with its cross-site embeddable shopping cart.
Publishers can include Shopbeam in a page to turn ads, images, or links into rich product widgets.
Visitors can click these products to see product details in a modal window, add to Shopbeam's cross-site shopping cart, and complete checkout without living the site."

File: README.md [hunk 2]
```diff
@@ -1,73 +1 @@
...
-####The Shopbeam Widget
-
-The Shopbeam 'widget' is the element that is embedded onto a hosting site's page or served over an ad network which the user can interact with to begin their on-site shopping experience. The Shopbeam widget can be either an html image made active with javascript or a flash swf. The instructions below are for creating and embedding a flash swf Shopbeam widget.
-
-Usage
------
-
-####Linking MovieClips to Shopbeam products
-
-In "YourDocumentClass" file instantiate the Shopbeam library with your API key. Once instantiated call the onClickGoToProduct function of your Shopbeam object passing the MovieClip name and product ID as arguments to wire up a MovieClip to the Shopbeam product lightbox like:
```
Comment (benjamine) [inline README.md:15]: maybe for example realism, invent our example publisher?, like ShopAndBeam/Acme/Microsoft's Northwind/etc.
 `YourDocumentClass` => `AcmeMovieClip`

File: README.md [hunk 3]
```diff
@@ -1,73 +1 @@
...
-public class YourDocumentClass extends MovieClip {
-  public function YourDocumentClass() { 
-    var shopbeam:Shopbeam = new Shopbeam("YOUR-API-KEY", this);
-    shopbeam.onClickGoToProduct("movieClip1", "product1-id"); /* The first argument is a MovieClip name, the second is a product ID */
-    shopbeam.onClickGoToProduct("movieClip2", "product2-id"); /* optional additional call to wire up another MovieClip */
-  }
-}
-```
-
-You can find the com.shopbeam Actionscript library source file in the example on this repo  here: https://raw.githubusercontent.com/yaplas/yaplas.github.io/master/com/shopbeam/Shopbeam.as.
```
Comment (benjamine) [inline README.md:29]: links to file in the same repo can be relative, like `com/shopbeam/Shopbeam.as`

File: README.md (hunk 1 above)
Comment (seththetech) [inline README.md:4]: > Shopbeam tools add e-commerce capabilities to sites across the web (eg. fashion blogs, magazines, brand sites) with its cross-site embeddable shopping cart. 

This sentence is vague. After reading it you still don't know how either a publisher or a shopper interfaces with the product which is the core of what the product is. Be careful of parenthesis, though they can be really useful they are kind of a hack, and in this case the contents are superfluous. 

'Shopbeam tools'? Would this be 'Shopbeam's tools'? I guess you can say 'Microsoft hardware' and that works. It feels wrong though. 

Here's another iteration let me know what you think:

> Shopbeam adds eCommerce capabilities to any website. A page element such as an image or flash movie, once Shopbeam enabled, becomes the entry point to an on-site shopping experience with product detail views, a cart, and a checkout all accessible without leaving the page.

File: README.md (hunk 2 above)
Comment (seththetech) [inline README.md:15]: Sounds good to me.

File: README.md (hunk 3 above)
Comment (seththetech) [inline README.md:29]: Easier to read for sure, will make that change.

File: README.md (hunk 1 above)
Comment (benjamine) [inline README.md:4]: that's nice! the only thing I miss somehow, is the fact the cart is a cross-site cart, follows the user across the web, but maybe that's not as important and can be omitted for brevity's sake.

File: README.md [hunk 4]
```diff
@@ -1,73 +1 @@
-Shopbeam Overview
------
-
-Shopbeam provides tools for advertisers and publishers to create and serve display advertisements which load product detail windows on-site. These on-publisher site product detail modal windows can be used to add products to a cross site shopping cart and purchase these products without clicking away from host/publisher website.
-
-####The Shopbeam Widget
-
-The Shopbeam 'widget' is the element that is embedded onto a hosting site's page or served over an ad network which the user can interact with to begin their on-site shopping experience. The Shopbeam widget can be either an html image made active with javascript or a flash swf. The instructions below are for creating and embedding a flash swf Shopbeam widget.
```
Comment (benjamine) [inline README.md:8]: suggestion: 
"
Shopbeam "widgets" are visual elements embedded in publisher sites or served over an ad network, users can interact with them to begin their on-site shopping experience.
Shopbeam widgets can be images or Flash movies ([SWF](http://en.wikipedia.org/wiki/SWF)).
The instructions below are for creating and embedding a custom Flash Shopbeam widget.
"

File: README.md (hunk 1 above)
Comment (seththetech) [inline README.md:4]: > Shopbeam adds eCommerce capabilities to any website. A page element such as an image or flash movie, once Shopbeam enabled, becomes the entry point to an on-site shopping experience with product detail views, a cart, and a checkout all accessible without leaving the page.
> 
> Once a product is added to a shopper’s cart, it is available to them across all Shopbeam enabled websites with a single checkout for items added from multiple sites.
> 
> Display advertisements are fully supported on Shopbeam enabled websites.

Comment (benjamine) [inline README.md:4]: :+1: , I wasn't aware that "Display advertisements" were a thing

**Pull request closed without merging**